apps/sms/views/conversation/js/conversation.js

```diff
@@ -2837,13 +2847,8 @@ var ConversationView = {
   },
 
   prompt: function conv_prompt(opt) {
-    var complete = (function complete() {
-      if (!Navigation.isCurrentPanel('thread')) {
-        Navigation.toPanel('thread', { id: Threads.currentId });
-      }
-    }).bind(this);
-
-    var thread = Threads.active;
+    var currentId = this.activeThread.id;
+    var thread = this.activeThread;
```

Comment:
nit: maybe reverse these 2 lines, so that you can have `var currentId = thread.id` ?